src/mmw/mmw/settings/layer_settings.py

```diff
@@ -145,6 +145,30 @@
         # overlaps with perimeter polygon.
     },
     {
+        'code': 'nhd_quality_tp',
+        'display': 'Delaware River Basin TP Concentration',
```

Comment:
I believe these should be labeled as for the "National" rather than "Delaware River Basin" data.